packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -729,8 +737,24 @@ export class Cluster extends ClusterBase {
       policy: AwsCustomResourcePolicy.fromSdkCalls({
         resources: AwsCustomResourcePolicy.ANY_RESOURCE,
       }),
+      installLatestAwsSdk: false,
```

Comment:
good catch!

Reply:
It's possible that it never did and that it was committed with `--dry-run`. It's also possible that this used to work when  it was committed and that it no longer works.